wss-agent-api/src/main/java/org/whitesource/agent/api/model/AgentProjectInfo.java

```diff
@@ -141,4 +142,8 @@ public String getViaLanguageName() {
     public void setViaLanguageName(String viaLanguageName) {
         this.viaLanguageName = viaLanguageName;
     }
+
+    public Collection<SecretFinding> getSecretFindings() {
```

Comment:
add the `setSecretFindings()` method as well

Reply:
done